conversation_reconstruction_dataflow/conversation_reconstruction/setup.py

```diff
@@ -62,13 +62,8 @@ class build(_build):  # pylint: disable=invalid-name
 #
 # The output of custom commands (including failures) will be logged in the
 # worker-startup log.
-CUSTOM_COMMANDS = [
-    ['apt-get', 'update'],
-    ['apt-get', '--assume-yes', 'install', 'p7zip-full'],
-    ['python', 'construct_utils/mwparserfromhell/setup.py', 'install'],
-    ['echo', 'Custom command worked!']]
-
-
+CUSTOM_COMMANDS = []
+#    ['python', 'construct_utils/mwparserfromhell/setup.py', 'install']]
```

Comment:
Here I commented it out because I wanted to test other parts, but this installation is necessary.
It got an exit code 2 when I was trying to set it up on dataflow pipeline, it worked fine when I was running locally.